packages/pwa-kit-runtime/src/utils/uuidv4.client.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * Simple implementation to create a uuid using crypto
+ * See: https://stackoverflow.com/questions/105034/how-do-i-create-a-guid-uuid
+ * @returns {*}
+ */
+export function uuidv4() {
+    return ([1e7] + -1e3 + -4e3 + -8e3 + -1e11).replace(/[018]/g, (c) =>
+        (c ^ (crypto.getRandomValues(new Uint8Array(1))[0] & (15 >> (c / 4)))).toString(16)
+    )
```

Comment:
Since we are sing the result of this function as the requestId internally, which was previously guaranteed to be unique since it was simply an incrementing number, we need to make sure that this function spits out unique values. Do we know if that is what happens here? Similarly with the server uuid util?

Reply:
Based on the fact that we are using `crypto.getRandomValues` here, I believe that function ensures a certain level of entropy that is acceptable for this useage.